src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+public class DukeException extends Exception {
+    /**
+     * Returns a dukeException.
```

Comment:
According to the coding standard provided, there should be an empty line between description and parameter section